src/entities/activity/ui/items/ItemPicker.tsx

```diff
@@ -38,6 +39,15 @@ export const ItemPicker = ({ item, onValueChange, isDisabled, replaceText }: Ite
         />
       );
 
+    case 'paragraphText':
+      return (
+        <ParagraphTextItem
+          item={item}
+          value={item.answer[0]}
+          onValueChange={onValueChange}
+          isDisabled={isDisabled}
+        />
+      );
```

Comment:
There's an unsafe assignment warning on this case.
Please add empty line between cases.